core/systems/home/view.ex

```diff
@@ -22,11 +18,153 @@ defmodule Systems.Home.View do
   def render(assigns) do
     ~H"""
     <div>
-      <Margin.y id={:page_top} />
-      <Area.content>
-        <.stack fabric={@fabric} gap="gap-14"/>
-      </Area.content>
+      <div class="bg-grey6">
+        <%= render_intro(assigns) %>
```

Comment:
Could you move these render functions to a seperate systems/home/html.ex file? This is the pattern I try to follow.

Reply:
See for example: Systems.Manual.Html